src/ui/components/Button/Button.jsx

```diff
@@ -0,0 +1,37 @@
+import { theme } from '../../../shared/theme';
+import ButtonSilverUrl from '../../../public/img/silverButton.png';
+import ButtonGoldUrl from '../../../public/img/goldButton.png';
+
+const BUTTON_HEIGHT_SIZE = '8rem';
+const BUTTON_WIDTH_SIZE = '21.25rem';
+const BUTTON_FONT_SIZE = '1.375rem';
+
+/**
+ * @param {{ children: React.ReactNode, type: 'silver' | 'gold', onClick: () => void }} props
+ */
+const Button = ({ children, type, onClick }) => {
+  return (
+    <button
+      type='button'
+      css={{
+        width: BUTTON_WIDTH_SIZE,
+        height: BUTTON_HEIGHT_SIZE,
+        background:
+          type === 'silver'
+            ? `url(${ButtonSilverUrl}) center center / cover`
+            : `url(${ButtonGoldUrl}) center center / cover`,
```

Comment:
```suggestion
        background: `url(${type === 'silver' ? ButtonSilverUrl : ButtonGoldUrl}) center center / cover`,
```

Możesz włożyć ternary głębiej żeby się nie powtarzać.